airflow/www/views.py

```diff
@@ -144,14 +144,6 @@
 )
 
 
-def truncate_task_duration(task_duration):
```

Comment:
This is removed outright since the function is not used at all in the entire code base.